README.md

```diff
@@ -1,201 +1,157 @@
+SensuFlow is a prescriptive monitoring as code workflow that uses the Sensu Go API and sensuctl commands to synchronize Sensu resources for multiple Sensu namespaces as part of a CI/CD system.
+Specifically, SensuFlow executes [`sensuctl create`][19] and [`sensuctl prune`][20] commands.
+Use SensuFlow to ensure that updates to the Sensu resources in your repository are reflected in your Sensu configuration.
 
+SensuFlow is implemented as a Bash shell script, `sensuflow.sh`, that executes sensuctl commands in a Docker container.
+SensuFlow includes linting logic to ensure your Sensu resource definitions are self-consistent with respect to your namespace directory structure and label-matching rule.
 
-## How It Works
-This Action provides an opinionated best practises to using the `sensuctl create` and `sensuctl prune` commands in order to efficiently manage Sensu monitoring resources in one or more namespaces. The Action automates several resource linting actions to help ensure self-consistent monitoring resources are defined prior to updating any Sensu resources.
+## Requirements
 
-This is achieved by processing a directory structure where each subdirectory is mapped to a Sensu namespace.
-By default, the required directory structure looks like:
-```
-.sensu/
-  cluster/
-    namespaces.yml
-  namespaces/
-    <namespace>/
-      checks/
-      hooks/
-      filters/
-      handlers/
-      handelersets/
-      mutators/
-  
-```
-where `<namespace>` is a placeholder for each Sensu namespace under management. The `cluster/` directory can be used to optionally manage Sensu cluster-wide resources, such as namespaces, if the Sensu RBAC profile in use allows for cluster-wide resource management.
-  
-## Setup
+The SensuFlow script requires these executables:
+
+- [jq][10]
+- [yq][11]
+- [sensuctl][12] (configured to communicate with the Sensu backend that you want to use SensuFlow with)
+
+To use SensuFlow, you'll also need:
+
+- A Sensu role-based access control (RBAC) [service account][13] with sufficient privileges to manage your monitoring configuration as code.
+- A [resource labeling][14] convention to designate which resources should be managed by SensuFlow.
+- A repository that contains Sensu monitoring code, organized in the correct [directory structure][15].
+- An integrated CI/CD system to run sensuctl commands using the Sensu service account from your Sensu monitoring code repository.
+
+A vendor-neutral `sensu/sensu-flow` [Docker container image][1] is available as of version `0.6.0`.
+You can use the Docker container image in your local development environment or adapt it to use with any CI/CD service that can use container images for CI/CD jobs.
+More information is available for the following CI/CD services: 
+
+- [Bitbucket][4]
+- [GitHub][5]
+- [GitLab][6]
+
+## Install SensuFlow
+
+**TODO**
```

Comment:
What do users need to clone/download/install to use SensuFlow in their own repos? Is it something like this:

1. Clone the SensuFlow repo (or reproduce it in Bitbucket or GitLab, if not using GitHub).
2. Add your namespace directories and resource files (with appropriate labels and following the prescribed directory structure).
3. Configure the Sensu RBAC service account.
4. Configure env vars required to use SensuFlow and aspects of the CI/CD service (secrets; pipeline files; other).

Anything else?

Is there a different install process for people who the Docker image?